toolkit/resources/assets/grub2/grub

```diff
@@ -0,0 +1,12 @@
+GRUB_TIMEOUT=5
+GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/os-release)"
+GRUB_DISABLE_LINUX_UUID=false
+GRUB_DEVICE_UUID="{{.BootUUID}}"
+GRUB_DEFAULT=saved
+GRUB_DISABLE_SUBMENU=true
+GRUB_TERMINAL_OUTPUT="console"
+GRUB_CMDLINE_LINUX="{{.LuksUUID}} {{.LVM}} {{.IMAPolicy}} {{.ReadOnlyVerityRoot}} {{.SELinux}} rd.auto=1 lockdown=integrity sysctl.kernel.unprivileged_bpf_disabled=1 {{.CGroup}} {{.ExtraCommandLine}}"
```

Comment:
Let's not do the toolkit-generated substitutions. In fact I don't think the toolkit should do anything to grub config at all, it should only invoke grub. The grub package should install the default config file (either in %files or as a postinst script action).